src/index.ts

```diff
@@ -3,9 +3,17 @@ import mongoose from 'mongoose';
 import dotenv from 'dotenv';
 import apiRoutes from './routes/api';
 import { init as initDiscord } from './discord';
+import { JWTPayload } from 'jose/webcrypto/types';
 
 import './util/keypair'; //Make sure pub/priv keygen is done
 
+declare module 'express-serve-static-core' {
```

Comment:
Used declaration merging to ensure Request can be passed in api without raising errors, lemme know if there's a more optimal way based on original design